airflow/www/static/js/dag/details/dagRun/MarkFailedRun.jsx

```diff
@@ -22,7 +22,7 @@ import { Button, useDisclosure } from '@chakra-ui/react';
 
 import { useMarkFailedRun } from '../../../api';
 import ConfirmDialog from '../ConfirmDialog';
-import { getMetaValue } from '../../../../utils';
+import { getMetaValue } from '../../../utils';
```

Comment:
In a later PR I will update more js/jsx files to typescript so we can remove more relative paths. I was having issues getting relative paths for js and ts to play nicely in our existing webpack configuration.